gradio/blocks.py

```diff
@@ -516,6 +517,7 @@ def __init__(
             js: Custom js or path to js file to run when demo is first loaded. This javascript will be included in the demo webpage.
             head: Custom html to insert into the head of the demo webpage. This can be used to add custom meta tags, scripts, stylesheets, etc. to the page.
             fill_height: Whether to vertically expand top-level child components to the height of the window. If True, expansion occurs when the scale value of the child components >= 1.
+            delete_cache: A tuple corresponding [frequency, age] both expressed in number of seconds. Every `frequency` seconds, the temporary files created by this Blocks instance will be deleted if more than `age` seconds have passed since the file was created. For example, setting this to (86400, 86400) will delete temporary files every day. The cache will be deleted entirely when the server restarts. If None, no cache deletion will occur.
```

Comment:
```suggestion
            delete_cache: A tuple corresponding [frequency, age] both expressed in number of seconds. Every `frequency` seconds, the temporary files created by this Blocks instance will be deleted if more than `age` seconds have passed since the file was created. For example, setting this to (86400, 86400) will delete all temporary files that are at least one day old, every day. Setting any non-None value will also cause the cache will be deleted entirely when the Gradio app is closed using the .close() method. If None, no cache deletion will occur.
```

Reply:
Note: as far as I could tell, the temporary files are only deleted when the server is closed using the .close() method, but not if the Python process is terminated with cmd+C for example

Reply:
Just re-tested and the files are deleted (as long as they are not used in examples)